gpg-expiration.py

```diff
@@ -30,7 +31,8 @@ def parse_args():
     parser.add_argument(
         'key_id',
         help='''GPG key ID to check.
-        This can be any valid search string that uniquely identifies a key.''',
+        This can be any valid search string that uniquely identifies a key.
+        Use "all" to check all public keys in the current keyring.''',
```

Comment:
> I would have preferred using "--all" as an optional argument, but wasn't sure how to make that work with the required key_id arg, without breaking backwards compatibility.

I think this is worth breaking backwards compatibility over! This isn't a highly used repo, and I'm not publishing it anywhere beyond this GitHub repo. I'm okay with saying folks can `git pull` and update their crontabs by hand.

In practice, I think that'd be done with [mutual exclusion](https://docs.python.org/3/library/argparse.html#mutual-exclusion), as you probably already found. That does come with some restraints, which I think you were alluding to, so I'd be happy with renaming the positional `key_id` argument to `--key-id`.

Would you be willing to add a mutual exclusion group with an `--all` argument, rename the positional argument, and update the documentation to match?

Reply:
Yep, happy to do that, in the next week or so when I have the chance.